### ISSUE TITLE
Luacontroller ignores pasted newlines

### ISSUE THREAD

**Issue by Bituvo** [minetest-mods/mesecons#682]
If I copy text that contains newlines into a luacontroller, all of the newlines disappear when I try to run it.

![image](https://github.com/user-attachments/assets/ce408543-9e00-4c88-8c7c-31533f15b215)

This problem occurs if I copy text from any source, including another properly-formatted luacontroller.

**Comment by SmallJoker** [comment]
@Bituvo Can you reproduce the same issue with other items, such as the Minetest Game book editor?

Copy&paste does not trigger any Lua callback on its own, thus this would be Luanti/Minetest engine issue.
Relevant code:
https://github.com/minetest-mods/mesecons/blob/dd073d2e83ef09475812a94f426c543ff89ae623/mesecons_luacontroller/init.lua#L690-L703

**Comment by Bituvo** [comment]
Yes, it does happen with the book as well. I'm using the stable release of 5.10.0, let me update to the latest commit...

**Comment by Bituvo** [comment]
I've updated to the latest commit and the issue still occurs; in the Luacontroller, all newlines are removed when I press "Execute" without having to re-open the formspec. With the book editor, the "Save" button saves a copy of the text with all newlines stripped from it.

**Comment by SmallJoker** [comment]
Please check for open Luanti/Minetest issues or open a new one if there's none.
Not a mesecons bug.

**Comment by Miniontoby** [comment]
Hmm, but that says it's merged, so what's up with that?
Or is it not yet in release?

**Comment by SmallJoker** [comment]
@Miniontoby I assume the issue reporter and you both use Windows, thus this should be fixed in 5.13.0 and newer. If it still occurs in said versions, then please open a new issue in the Luanti repo.

**Comment by Miniontoby** [comment]
Yeah, that's right, as I supplied all the info in the original issue that I made which redirected me to this one: https://github.com/minetest-mods/mesecons/issues/712

And I was using 5.12.0, so yeah I'll just update to 5.13.0 I guess
